scripts/generate-release-cue.rb

```diff
@@ -145,16 +145,13 @@ def generate_changelog!(new_version)
     contributors = Array.new
 
     if last.start_with?("authors: ")
-      authors_str = last[9..]
-      authors_str = authors_str.delete(" \t\r\n")
-      authors_arr = authors_str.split(",")
-      authors_arr.each { |author| contributors.push(author) }
+      contributors = last[9..].split(" ").map(&:strip)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏this is interesting. I saw the `check_changelog_fragments` script is only looking for the presence of the "authors:" at the beginning of the line, but doesn't validate how it is delimited. We could consider adding that validation to the script to ensure it is correct. 

Separately, I think the README.md for the feature needs to be updated (because it does reference comma delimiter)

Reply:
Good thoughts. I updated the readme and the changlog checker to catch comma delimited names in [61594b3](https://github.com/vectordotdev/vector/pull/20075/commits/61594b337b84acf42dd7e3567ecec034da721d0b)

Reply:
Thanks!